src/scroll.cc

```diff
@@ -200,9 +201,10 @@ void Scroll::clear() {
     if (!impl->isScrollStarted()) {
         LOG(LogLevel::INFO, "There is no scroll started (scrollId is empty).");
     } else {
+        const std::string requestBody{"{\"scroll_id\": \"" + scrollParameters.scrollId + "\"}"};
```

Comment:
```suggestion
        const std::string requestBody{"{\"scroll_id\": [\"" + scrollParameters.scrollId + "\"]}"};
```
Array works even with historical version (2.3) of elastic cluster.

Reply:
Should be fixed now.